### PR TITLE
Add an 'ns' shortcut, tidy capital letters

### PR DESCRIPTION
**Pull request by thockin**
Not 1.0 critical

@jlowdermilk

### CODE REVIEW

**Comment by jlowdermilk** [comment]
lgtm. Would be safe to add, but not worth cherrypick.

File: pkg/kubectl/kubectl.go [hunk 1]
```diff
@@ -108,14 +108,14 @@ func expandResourceShortcut(resource string) string {
 		"ep":     "endpoints",
 		"limits": "limitRanges",
```
Comment (nikhiljindal) [inline pkg/kubectl/kubectl.go]: While you are at it, tidy this one as well?

Comment (thockin) [inline pkg/kubectl/kubectl.go]: missed that, done.

**Comment by erictune** [comment]
I'm not merging any code until Jenkins is green for a turn.

**Comment by thockin-cc** [comment]
This is supposed to be v1-post. No need to merge this until we open the
gate.

**Comment by bgrant0607** [comment]
LGTM
cc @hurf

**Comment by hurf** [comment]
@bgrant0607 I'll make related changes after this patch is merged.

**Comment by vishh** [comment]
@k8s-bot: ok to test

File: pkg/kubectl/kubectl.go [hunk 2]
```diff
@@ -106,16 +106,16 @@ func expandResourceShortcut(resource string) string {
...
-		"pvc":    "persistentVolumeClaims",
-		"quota":  "resourceQuotas",
+		"pv":     "persistentvolumes",
+		"pvc":    "persistentvolumeclaims",
+		"quota":  "resourcequotas",
 		"rc":     "replicationcontrollers",
-		// DEPRECATED: will be removed before 1.0
-		"se":  "services",
-		"svc": "services",
+		"se":     "services", // DEPRECATED: will be removed before 1.0
```
Comment (bgrant0607) [inline pkg/kubectl/kubectl.go]: Please remove the deprecated comment. We forgot to remove it, so we're stuck with it.

**Comment by bgrant0607** [comment]
LGTM.

**Comment by thockin** [comment]
You don't think we can get rid of it?  We have a bunch of things that were
left until we crossed the 1.0 threshold.

**Comment by bgrant0607** [comment]
Did we print a deprecated warning or anything? We can't expect users to look at the code.

**Comment by thockin** [comment]
I don't think it is documented anywhere, but I also have not scanned
examples for use of it (out of the house now).  If no examples use it and
it is not documented...

**Comment by thockin** [comment]
It's not in any examples and it is not documented.  I think we can kill
it.  I'll send a PR and we can debate.

**Pull request merged**